community/kernel-api/src/test/java/org/neo4j/internal/kernel/api/SchemaReadWriteTestBase.java

```diff
@@ -46,7 +46,7 @@
 @SuppressWarnings( "Duplicates" )
 public abstract class SchemaReadWriteTestBase<G extends KernelAPIWriteTestSupport> extends KernelAPIWriteTestBase<G>
 {
-    private int label, type, prop1, prop2;
+    private int label, label2, type, prop1, prop2, prop3;
```

Comment:
Did not know we allowed multi-variable declarations.

Reply:
You're welcome :)

Reply:
:see_no_evil: